plugins/externalwrench/include/gazebo/ApplyExternalWrench.hh

```diff
@@ -27,22 +29,26 @@ public:
     virtual void        threadRelease();
     yarp::os::Bottle    getCmd();
     void                setRobotName(std::string robotName);
-    void                setScopedName(std::string scopedName);
-    void                setDefaultLink(const std::string& defaultLink);
-    void                setDurationBuffer(double d);
-    double              getDurationBuffer();
+    void                setRobotModel(physics::ModelPtr robotModel);
     void                setNewCommandFlag(std::int32_t flag);
     virtual void        onStop();
+
+    boost::shared_ptr< std::vector< boost::shared_ptr<ExternalWrench>>> wrenchesVectorPtr{new std::vector< boost::shared_ptr<ExternalWrench>>()};
```

Comment:
Why can't we use a good old `std::vector<ExternalWrench>`?

Reply:
Similar to https://github.com/robotology/gazebo-yarp-plugins/pull/418#discussion_r273440396